p2p/src/main/java/bisq/network/p2p/network/TorNetworkNode.java

```diff
@@ -249,7 +253,7 @@ private void createTorAndHiddenService(File torDir, int localPort, int servicePo
                 Torrc override = null;
 
                 // check if the user wants to provide his own torrc file
-                String torrcfile = System.getProperty("torrcfile");
+                String torrcfile = System.getProperty(CONFIG_TORRCFILE);
```

Comment:
Should we use our program arguments instead of system properties? I am not sure whats better here or if both make sense. Just want to avoid that we mix it as we never use system properties for application specific options so far. But maybe it integrates better with certain environments like with Tails?

Reply:
I thought about that but decided that the implementation is much faster when done this way. I have not been aware of the historical ways of doing it.
I will look into it.

Reply:
You can find it in NetworkOptionKeys. It is a bit cumbersome to define all but if you follow one existing example its not hard. I would prefer to stick with program arguments as they also come with the help option.